src/rust/engine/process_execution/src/lib.rs

```diff
@@ -370,6 +370,20 @@ impl ProcessResultMetadata {
   pub fn new(total_elapsed: Option<Duration>) -> Self {
     ProcessResultMetadata { total_elapsed }
   }
+
+  /// How much faster a cache hit was than running the process again.
+  ///
+  /// This includes the overhead of setting up and cleaning up the process for execution, and it
+  /// should include all overhead for the cache lookup.
+  pub fn time_saved_from_cache(
+    &self,
+    cache_lookup: std::time::Duration,
+  ) -> Option<std::time::Duration> {
```

Comment:
This file already imports `Duration` (and in general, it's a very common type), so the absolute import shouldn't be necessary.

Reply:
`conrete_time::Duration` vs. `std::time::Duration`